lti_emailer/settings/unit_test.py

```diff
@@ -14,6 +14,12 @@
     },
 }
 
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.dummy.DummyCache'
+    },
+}
+
 """
 Unit tests should be able to run without talking to external services like
 redis/elasticache.  Let's disable the caching middleware, and use the db
```

Comment:
BTW, all the stuff below this is only needed if you're using LiveServerTestCase, which you don't seem to be.